Test/dafny0/ArrayElementInit.dfy

```diff
@@ -73,8 +73,8 @@ method Q0(s: Six, y: int) returns (a: array<Six>)
 {
   if
   case true =>  a := new Six[10](_ => s);
-  case true =>  a := new Six[10](x => 6+x);
-  case true =>  a := new Six[10](_ => y);
+  case true =>  // a := new Six[10](x => 6+x); // type error
+  case true =>  // a := new Six[10](_ => y);   // type error
```

Comment:
I don't expect these to be (type or verification) errors.

Reply:
OK - but will need checking in the verifier